src/app/shared/components/route-selector/route-selector.component.ts

```
@@ -100,7 +103,7 @@ export class RouteSelectorComponent implements OnInit, ControlValueAccessor, OnD
   }
 
   customDistanceValidator(control: AbstractControl): { invalidDistance: boolean } {
-    const passedInDistance = parseFloat(control.value);
+    const passedInDistance = parseFloat(String(control.value));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)